lib/capybara/screenshot/diff/vcs.rb

```diff
@@ -9,7 +9,7 @@ module Vcs
 
         def restore_git_revision(name, target_file_name)
           redirect_target = "#{target_file_name} #{SILENCE_ERRORS}"
-          show_command = "git show HEAD~0:./#{Capybara::Screenshot.screenshot_area}/#{name}.png"
+          show_command = "git show HEAD~0:./#{Capybara::Screenshot.screenshot_area_abs}/#{name}.png"
```

Comment:
Are you sure this is right?

Reply:
the problem that it tries to load `doc/screenshots` where `Rails.root` is not git root. So I just review code below, and it use abs path

Reply:
or we need to cd to the `root` before to run this command

Reply:
But when you use the `_abs` path, you should not prefix with "./", right?

Reply:
@donv I think I generated bad Rails.root, that's why I need to have `./` I'll recheck once more

Reply:
and will try to add tests for this

Reply:
@donv reverted my code, please recheck